akka-http2-support/src/test/scala/akka/http/scaladsl/Http2ServerTest.scala

```diff
@@ -37,13 +38,8 @@ object Http2ServerTest extends App {
     akka.http.server.preview.enable-http2 = true
                                                    """)
   implicit val system = ActorSystem("ServerTest", testConf)
-  import system.dispatcher
-
-  val settings = ActorMaterializerSettings(system)
-    .withFuzzing(false)
-    //    .withSyncProcessingLimit(Int.MaxValue)
-    .withInputBuffer(128, 128)
-  implicit val fm = ActorMaterializer(settings)
+  implicit val ec: ExecutionContext = system.dispatcher
+  implicit val fm = ActorMaterializer()
```

Comment:
I guess here it's still needed because we materialize streams which needs an explicit implicit materializer until 2.5.31. :+1:

Reply:
Hmm, I just tested locally replacing `ActorMaterializer()` with `SystemMaterializer(system).materializer` and works just fine. Do we really need the `ActorMaterializer()`?

Reply:
Indeed an intermediate solution might be to use SystemMaterializer here, but we'd really want to remove the line - let's see if with 2.5.31 we can.